docs/en_US/PaiMode.md

```diff
@@ -59,6 +59,8 @@ Compared with LocalMode and [RemoteMachineMode](RemoteMachineMode.md), trial con
     * Optional key. Set the virtualCluster of OpenPAI. If omitted, the job will run on default virtual cluster.
 * shmMB
     * Optional key. Set the shmMB configuration of OpenPAI, it set the shared memory for one task in the task role.
+* gpuType
+    * Optional key. Specify the GPU type to be used in the tasks. If omitted, the job will run on any gpu type
```

Comment:
Can we list some optional choice here...

Reply:
Sure. However, nni 179 cluster only use K80 gpuType, according to [pai issue](https://github.com/microsoft/pai/blob/master/docs/hadoop-ai/README.md#known-issues) gpuType will work after manually updating the configuration on PAI.